pkg/log/zap/zap.go

```diff
@@ -149,6 +137,48 @@ func (o *Options) addDefaults() {
 	o.ZapOpts = append(o.ZapOpts, zap.AddStacktrace(o.StacktraceLevel))
 }
 
+// Development sets the Development option for the logger
+func Development(development bool) func(o *Options) {
+	return func(o *Options) {
+		o.Development = development
+	}
+}
+
+// Encoder sets the Encoder option for the logger
```

Comment:
Should we probably mention the defaults in the godocs for the new funcs?

Reply:
We have them defined already for the Option fields so I wasn't sure if I should repeat those here again.
https://github.com/kubernetes-sigs/controller-runtime/blob/16c93b06b0a8cb99e002529cb1b43dce5e489baf/pkg/log/zap/zap.go#L93-L94